### PR TITLE
#284 - Added 'Cucumber Can't Find my Step Definitions in IntelliJ IDEA'

### PR DESCRIPTION
**Pull request by helenjoscott**
Added another FAQ. Still having Docker issues so marking as WIP for automation to run.

### CODE REVIEW

**Comment by helenjoscott** [comment]
I used instructions from JetBrains and linked to the content that I believe will address this issue based on the description in the FAQs. Is that okay?

**Comment by helenjoscott** [comment]
Added the final FAQ for transitive dependencies in Cucumber

File: content/docs/community/faq.md [hunk 1]
```diff
@@ -253,3 +253,21 @@ definitions, as that will lead to duplicates.
...
+In this instance, you need to configure a new run configuration in IntelliJ IDEA. 
+1. Click **Run** > **Edit Configurations** from the menu in IntellJ IDEA.
+2. Click the **+** icon on the top-left and type in _cucumber_. Select **Cucumber Java**.
+3. Create the configuration according to the [Run/Debug Configuration Cucumber Java](https://www.jetbrains.com/help/idea/run-debug-configuration-cucumber-java.html) instructions from JetBrains. 
+   
+If IntelliJ IDEA doesn't recognize the package with step definitions, you can specify it manually by entering the package name in the Glue field, for example _stepdefs_.
+
+For more information, please see [Run Cucumber Tests](https://www.jetbrains.com/help/idea/running-cucumber-tests.html) from JetBrains documentation.
+
+# Running Cucumber results in a stack trace containing 'Failed to instantiate public cucumber.runtime.java.JavaBackend or NoSuchMethodException'
```
Comment (luke-hill) [inline content/docs/community/faq.md]: This one is a bit ambiguous from the POV of what is my question e.t.c.

Is this `Cucumber JVM Throws Error with a stacktrace containing .....`

Comment (helenjoscott) [reply content/docs/community/faq.md]: Hi @luke-hill, I don't know. I've used what information there was on the ticket (https://github.com/cucumber/docs.cucumber.io/issues/284) but I'm not sure what else I can add. If anyone has more information I can absolutely add it though.

Comment (luke-hill) [reply content/docs/community/faq.md]: Sorry I mean can you frame this as a Frequently Asked question. It's a bit ambiguous at the moment if this is a problem or solution. I had a brief look at the issue you linked, so I think you should re-phrase this as.

> How do I fix Cucumber-JVM erroring with a stacktrace 'Failed to instantiate public cucumber.runtime.java.JavaBackend or NoSuchMethodException'?

Comment (helenjoscott) [reply content/docs/community/faq.md]: Ahhh, okay, was being slow :D will do!

**Comment by luke-hill** [comment]
Looks good. There's some minor things we could maybe do here, like wrap this in a jvm block so it only shows to relevant users. But given I understand what this PR does, and it's a first PR, it's still an improvement.

I'll let someone else approve and merge as I'm a bit removed from the docs for stuff like this now :) Well done though

**Comment by helenjoscott** [comment]
@mlvandijk are you able to review this from a doc POV or do you know someone who can?

**Review by luke-hill — APPROVED** [review]

**Comment by luke-hill** [comment]
Given this is all new content from a previous issue I'll just approve and merge.

**Pull request merged**